src/main/java/com/bunq/sdk/context/ApiContext.java

```diff
@@ -228,7 +225,7 @@ public boolean ensureSessionActive() {
 
   public boolean isSessionActive() {
     return sessionContext != null &&
-      getTimeToSessionExpiryInSeconds() < TIME_TO_SESSION_EXPIRY_MINIMUM_SECONDS;
+      getTimeToSessionExpiryInSeconds() > TIME_TO_SESSION_EXPIRY_MINIMUM_SECONDS;
```

Comment:
@OGKevin Just to be sure are you sure that this won't create any issues?

Reply:
according to my testing no, It was recreating a session on every call 😅 now it seems to behave as expected. Pipeline should not fail with 429 errors on each test anymore.